client/layout/masterbar/masterbar-cart.tsx

```diff
@@ -0,0 +1,118 @@
+import { Popover } from '@automattic/components';
+import { CheckoutProvider, CheckoutErrorBoundary, Button } from '@automattic/composite-checkout';
+import { useShoppingCart } from '@automattic/shopping-cart';
+import { useTranslate } from 'i18n-calypso';
+import page from 'page';
+import React, { useRef, useState } from 'react';
+import CalypsoShoppingCartProvider from 'calypso/my-sites/checkout/calypso-shopping-cart-provider';
+import { CheckoutSummaryTotal } from 'calypso/my-sites/checkout/composite-checkout/components/wp-checkout-order-summary';
+import { WPOrderReviewLineItems } from 'calypso/my-sites/checkout/composite-checkout/components/wp-order-review-line-items';
+import MasterbarItem from './item';
+
+import './masterbar-cart-style.scss';
+
+type MasterbarCartProps = { selectedSiteSlug: string | undefined };
+
+export function MasterbarCart( { selectedSiteSlug }: MasterbarCartProps ): JSX.Element | null {
+	const { responseCart, reloadFromServer } = useShoppingCart( selectedSiteSlug );
+	const masterbarButtonRef = useRef( null );
+	const [ isActive, setIsActive ] = useState( false );
+	const translate = useTranslate();
+
+	if ( ! selectedSiteSlug || responseCart.products.length < 1 ) {
+		return null;
+	}
+
+	const onClick = () => {
+		setIsActive( ( active ) => {
+			if ( ! active ) {
+				reloadFromServer();
+			}
+			return ! active;
+		} );
+	};
+	const onClose = () => setIsActive( false );
+	const tooltip = translate( 'View my Shopping Cart' );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **34** times:
`translate( 'My Shopping Cart' )` *ES Score: 7*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/try/masterbar-cart-app)